modules/webgl/src/adapter/resources/webgl-render-pass.ts

```diff
@@ -58,7 +55,7 @@ export class WEBGLRenderPass extends RenderPass {
    * Maps RenderPass parameters to GL parameters
    */
   setParameters(parameters: RenderPassParameters = {}): void {
-    const glParameters: GLParameters = {};
+    const glParameters: GLParameters = {...this.glParameters};
```

Comment:
Note that only four specifically documented parameters can be changed after creation in a RenderPass (blend constant, stencil reference, scissor rect and viewport). Other parameters are frozen. 

We should not be touching framebuffer, writeMask, stencilMask etc here.